packages/cnpj-dv/src/CnpjCheckDigits.php

```
@@ -144,75 +168,73 @@ private function parseInput(mixed $cnpjInput): array
-    private function parseArrayInput(array $cnpjArray): array
+    private function parseArrayInput(array $cnpjArray): string
     {
+        $buffer = '';
+
         if ($cnpjArray === []) {
-            return [];
+            return $buffer;
         }
 
         foreach ($cnpjArray as $item) {
             if (!is_string($item)) {
```

PHPStan (2.1.54)

[warning] Call to function is_string() with string will always evaluate to true.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(function.alreadyNarrowedType)